### ISSUE TITLE
item taking two columns

### ISSUE THREAD

**Issue by ceehaupt** [bigbitecreative/macy.js#77]
Hi,

I just want to ask if it is possible with macy to have some items with a width of two columns like in isotope https://codepen.io/desandro/pen/mEinp ?

Best

**Comment by hrishikesh-k** [comment]
@ceehaupt 

I am probably too late to answer, but, I just came here after implementing it on my website and had the same question. But, what a bummer. According to this issue https://github.com/bigbite/macy.js/issues/35, it's not possible and might never get implemented.

So, I gotta look for another library now, which is kinda disappointing as Macy had otherwise worked perfectly for me.

**Comment by hrishikesh-k** [comment]
Update, I just thought of a very simple solution and it works great for me. I added a child container to the one that's getting sized by Macy and set its width to 200%. Done.

**Comment by riadcitaku** [comment]
@Hrishikesh-K is that being calculated? Mine is overflowing into the next item.

**Comment by hrishikesh-k** [comment]
@riadcitaku Yeah, it'll overflow into the next item. This library isn't designed to handle some item consuming 2 columns, so, the overflow is gonna happen and the next items aren't going to automatically adjust. It had worked in my case because the one I needed in 2 columns was supposed to be the last item in the row.
